pkg/util/hvutil/hvutil.go

```diff
@@ -23,3 +24,33 @@ func GetPodNamespace(annotations map[string]string) string {
 
 	return annotations[cri.SandboxNamespace]
 }
+
+func sanitize(input string) string {
+
+	var output string
+
+	for _, c := range strings.ToLower(input) {
+		if !(('a' <= c && c <= 'z') || ('0' <= c && c <= '9') || c == '-') {
+			c = '-'
+		}
+		output += string(c)
+	}
+
+	return output
+}
+
+func GetInstanceName(nodeName, podNamespace, podName, sandboxID string) string {
```

Comment:
nit: technically this is creating the name of the instance, so should this method be instead called `MakeInstanceName` or `CreateInstanceName` ?

Reply:
OK. I'll rename to `CreateInstanceName`